ReactGoStat/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "react-go-stat",
+  "version": "0.1.0",
+  "engines": {
+    "node": "14.15.0"
+  },
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "dependencies": {
+    "@geist-ui/react": "latest",
+    "@nektariosfifes/practical-react-ui-kit": "npm:@NektariosFifes/practical-react-ui-kit@^1.0.1",
```

Comment:
Generally, it is highly discouraged to use custom external libraries unless there is a good reason to do so. Why are you using a private library?

Reply:
yes this was a packaged version i was testing out i will remove it

Reply:
sry i by accident hit request review thinking it was refresh.
 I dont understand i pushed to my fork the correction yet i dont know if the pull has been modified